From c23d28bd5e8adf7186bc393a50ec784fc3b2528c Mon Sep 17 00:00:00 2001
From: Inaky Perez-Gonzalez <inaky.perez-gonzalez@intel.com>
Date: Tue, 23 Nov 2010 15:52:18 -0800
Subject: [PATCH 06/20] wimaxcu: fix array size

Array size declaration for 'cmd_args' was of the wrong size, thus
throwing a warning.

Reported-by: Andrey Kononov <kononov@zarya-k.ru>
Signed-off-by: Inaky Perez-Gonzalez <inaky@linux.intel.com>
---
 .../OSDependent/Linux/wimaxcu/wimaxcu_defs.h       |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/InfraStack/OSDependent/Linux/wimaxcu/wimaxcu_defs.h b/InfraStack/OSDependent/Linux/wimaxcu/wimaxcu_defs.h
index aa6df57..534848b 100644
--- a/InfraStack/OSDependent/Linux/wimaxcu/wimaxcu_defs.h
+++ b/InfraStack/OSDependent/Linux/wimaxcu/wimaxcu_defs.h
@@ -210,7 +210,7 @@ struct cu_cmd_specs
 
 	char *default_arg;	// eg. scan [preferred]
 
-	cmd_arg_map cmd_args[6];	// Maximum number of arguments any of cmds can accept (un-used ones will be either NULL or empty strings)
+	cmd_arg_map cmd_args[7];	// Maximum number of arguments any of cmds can accept (un-used ones will be either NULL or empty strings)
 
 	size_3_string_array scan_connect_mode_options[2];
 	char *cmd_mode_options[2];
-- 
1.7.4.1

